app/test/e2e/src/test-server.js

```diff
@@ -0,0 +1,18 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+const chai = require('chai');
+// eslint-disable-next-line import/no-unresolved
+const chaiHttp = require('chai-http');
+const app = require('../../../src/app');
+
+chai.use(chaiHttp);
+
+const createRequest = (prefix, method) => {
```

Comment:
could you explain the advantage of this vs using the approach already present on all other microservices?